dashboard/static/js/assign-progress.js

```diff
@@ -1,219 +1,200 @@
-var makeGraph = function () {
+jQuery(document).ready(function ($) {
```

Comment:
Should the javascript for displaying file access also follow the same pattern?

Reply:
I think the purpose of `$(document).ready()` to wait to do our stuff until the DOM is fully loaded. So maybe the code that is in there wait and do it supposed to be doing. It doesn't hurt to wrap the code in the `$(document).ready)`